builder/confd/confd.go

```diff
@@ -0,0 +1,95 @@
+// Package confd provides basic Confd support.
+//
+// Right now, this library is highly specific to the needs of the present
+// builder. Because the confd library is not all public, we don't use it directly.
+// Instead, we invoke the CLI.
+package confd
+
+import (
+	"github.com/Masterminds/cookoo"
+	"github.com/Masterminds/cookoo/log"
+	"github.com/Masterminds/cookoo/safely"
+
+	"fmt"
+	"os/exec"
+	"strconv"
+	"time"
+)
+
+// defaultEtcd is the default Etcd host.
+const defaultEtcd = "127.0.0.1:4001"
+
+// RunOnce runs the equivalent of `confd --onetime`.
+//
+// This may run the process repeatedly until either we time out (~20 minutes) or
+// the templates are successfully built.
+//
+// Importantly, this blocks until the run is complete.
+//
+// Params:
+// - node (string): The etcd node to use. (Only etcd is currently supported)
+//
+// Returns:
+// - The []bytes from stdout and stderr when running the program.
+//
+func RunOnce(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
+
```

Comment:
This isn't much of a concern. We let the tooling determine the acceptable styling of the code. As long as it passes `make test-style` (which in turn runs `gofmt`), we're happy with the styling.